src/client/components/input/entity-search.js

```diff
@@ -18,7 +18,8 @@
  */
 
 import {entityToOption, genEntityIconHTMLElement} from '../../helpers/entity';
-import {Async} from 'react-select';
+// eslint-disable-next-line import/no-internal-modules
```

Comment:
Let's configure the rule in `/.eslintrc.js` to allow imports from `**/react-select/*` - see example here:

https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-internal-modules.md